doc/metrics.md

```diff
@@ -0,0 +1,352 @@
+# Metrics
+
+This quick start is for end users of OpenTelemetry who wish to manually measure their applications. If you are a library author, please see the [Library Authors Guide](library-author.md). If you wish to automatically instrument your application, see the automatic instrumentation documentation for the SDK you wish to use.
+
+For a high-level overview of OpenTelemetry metrics in general and definitions of some common terms, you can refer to the [OpenTelemetry Specification Overview][spec-overview]
+
+_Metrics API Specification: <https://github.com/open-telemetry/opentelemetry-specification/blob/v1.14.0/specification/metrics/api.md>_
+
+_Metrics API Reference: <https://open-telemetry.github.io/opentelemetry-js-api/classes/metricseapi.html>_
+
+- [Acquiring a Meter](#acquiring-a-meter)
```

Comment:
I think we should add a "quick start" at the top which shows the minimum to set up the API and SDK and export to something (prom or console i guess). Right now I miss a full e2e example.

Reply:
Yep, having a "right way" to set up the SDK would be ideal. I can take that + what's here and more or less plant it in the website.

Re: a full e2e quick start ... perhaps, if there's a way to include that here and base it off of the [python quickstart](https://opentelemetry.io/docs/instrumentation/python/getting-started/) that could be cool. I plan on eventually reworking the Node quickstart in the docs to follow this pattern. So long as I know the best way to initialize the SDK/API for metrics, I can just take that on though.

Reply:
I will work on a getting started based on the Python one as first chapter of the file.

Reply:
I have added something @cartermp @dyladan